src/proc.c

```diff
@@ -290,6 +290,27 @@ int adjust_threads(int *tids, int nr_tids, int *user_tids,
     return 0;
 }
 
+int filter_threads(int tids[], int index[], char states[], int nr_tids,
+        const char user_states[])
```

Comment:
`const char *user_states` since it's actually NUL-terminated _string_.

Reply:
Fixed